firestore/src/appsscript.json

```diff
@@ -12,10 +12,10 @@
   "dataStudio": {
     "name": "Google Cloud Firestore",
     "logoUrl": "http://i66.tinypic.com/2mevmeq.png",
-    "company": "Google",
+    "company": "Google Data Studio Developer Relations",
     "companyUrl": "https://firebase.google.com/products/firestore/",
```

Comment:
We shouldn't have identical company names point to different company URLs. So In this case either remove companyUrl or make consistent with others: https://developers.google.com/datastudio/ or change company name to reflect company URL